src/modules/launcher/Plugins/Microsoft.Plugin.Indexer/Languages/en.xaml

```diff
@@ -8,5 +8,7 @@
     <system:String x:Key="Microsoft_plugin_indexer_open_in_console">Open path in console (Ctrl+Shift+C)</system:String>
     <system:String x:Key="Microsoft_plugin_indexer_name">Name</system:String>
     <system:String x:Key="Microsoft_plugin_indexer_path">Path</system:String>
-
+    <system:String x:Key="Microsoft_plugin_indexer_drivedetectionwarning">All drives may not be indexed, click to open Search Settings.</system:String>
```

Comment:
Text improvement suggestion:
`Maybe not all drives are indexed. Click to open Search Settings.`